### PR TITLE
Release/v3.15.0

### PR DESCRIPTION
**Pull request by sshugsc**
### Improvements:
- Add support for node 18, 20
- Remove support for node 12, 14
- KBDEV-1128: Update to orientjs v3.2

### Bug Fixes:
- KBDEV-1189:  Fix redoc issue
- KBDEV-858: Enable deleted records and statements to be viewed

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
         "jsonwebtoken": "^8.2.0",
         "lodash": "^4.17.21",
         "morgan": "^1.10.0",
-        "orientjs": "^3.0.11",
+        "orientjs": "^3.2.0",
```
Comment (mathieulemieux) [inline package-lock.json:25]: @sshu, do these changes to the package-lock.json file has been made manually or by running npm install?

Comment (sshugsc) [reply package-lock.json:25]: yes

Comment (mathieulemieux) [reply package-lock.json:25]: yes what? lol. manually?

Comment (sshugsc) [reply package-lock.json:25]: sorry, I mean it is made automatically by running npm install. This change came from https://github.com/bcgsc/pori_graphkb_api/pull/68

Comment (mathieulemieux) [reply package-lock.json:25]: ok ty!

**Review by mathieulemieux — APPROVED** [review]
One test keep failling because of GitHub timeout. I suggest we merge anyways at this time

**Pull request merged**